views/manager/_form.php

```diff
@@ -19,12 +19,15 @@
 echo $form->field($model, 'published')->checkbox();
 
 $settings = [
-    'lang' => Yii::$app->language,
     'minHeight' => 200,
     'plugins' => [
         'fullscreen',
     ],
 ];
+if (!in_array(Yii::$app->language, ['en', 'en_US', 'en-US'])) {
```

Comment:
Hummm, I don't think it's completely correct. I.e. I know locale en_GB :)
I don't know what is the difference, but it exists :)